.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,37 @@
+name: "Dependabot Scan"
+
+on:
+  workflow_call:
+
+jobs:
+  dependency-review-pr:
+    if: ${{github.event_name == 'pull_request'}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout repository'
+        uses: actions/checkout@v4
+
+      - name: 'Dependency review'
```

Comment:
This name is a bit vague, could we change it to something that makes it more obvious what's being reviewed? Same for the one below.